internal/service/common/common.go

```diff
@@ -43,6 +44,25 @@ func ResourceFromPb(resourceType, reporterId string, resourceData model.JsonObje
 	}
 }
 
+func ResourceFromPb(resourceType, reporterId string, resourceData model.JsonObject, workspaceId string, reporter *pbresourcev1beta2.ReporterData) *model.Resource {
+	return &model.Resource{
+		ID:           uuid.UUID{},
+		ResourceData: resourceData,
+		ResourceType: resourceType,
+		WorkspaceId:  workspaceId,
+		Reporter: model.ResourceReporter{
+			Reporter: model.Reporter{
+				ReporterId:      reporterId,
+				ReporterType:    reporter.ReporterType,
+				ReporterVersion: reporter.ReporterVersion,
+			},
+			LocalResourceId: reporter.LocalResourceId,
+		},
```

Comment:
We may need to flatten the Reporter data in the mapper here. In the multi-reporter PR I flattened ReporterResourceID (LocalResourceID) and ReporterID (ReporterType). See the `ResourceFromPbv1beta1` function. Those changes do not look reflected here.